das/daser.go

```diff
@@ -12,10 +12,8 @@ import (
 
 var log = logging.Logger("das")
 
-// TODO(@Wondertan): Initialization and warm-up.
-// TODO(@Wondertan): Start and Stop is better be thread-safe.
-// TODO(@Wondertan): Add some testing
 // DASer continuously validates availability of data committed to headers.
+// TODO(@Wondertan): Start and Stop is better be thread-safe.
 type DASer struct {
```

Comment:
Orthogonal to this PR: The name DASer is always a bit confusing to me. Let's think of a better name long term.

Reply:
@liamsi, totally agree. I was thinking about `da.Sync`, as essentially, it performs syncing of available shares. This component can be reused for FullSync of shares or Namespace sync, depending on shares.Availability implementation given to him.

Reply:
Let's keep it for now. And change the name later.